claat/render/template.md

```diff
@@ -1,10 +1,21 @@
+---
+id: {{.Meta.ID}}
+{{if .Meta.Status}}status: {{.Meta.Status}}{{end}}
+{{if .Meta.Summary}}summary: {{.Meta.Summary}}{{end}}
+{{if .Meta.Author}}author: {{.Meta.Author}}{{end}}
+{{if .Meta.Categories}}categories: {{commaSep .Meta.Categories}}{{end}}
+{{if .Meta.Tags}}tags: {{commaSep .Meta.Tags}}{{end}}
+{{if .Meta.Feedback}}feedback link: {{.Meta.Feedback}}{{end}}
+{{if .Meta.GA}}analytics account: {{.Meta.GA}}{{end}}
```

Comment:
I wonder if we should just always output these regardless of whether they're empty.
Then it's 1) easier to ready and 2) don't need to remember a field name when modifying existing markdown codelab.

But I'm ok with either. Whatever you prefer.

Reply:
Will leave it for now since we don't have tests to see how the parser deals with empty input on the metadata.